src/syscall/syscall_libc.go

```diff
@@ -193,6 +274,17 @@ func libc_mkdir(pathname *byte, mode uint32) int32
 //export rmdir
 func libc_rmdir(pathname *byte) int32
 
+// The odd $INODE64 suffix is an Apple compatibility feature, see https://assert.cc/posts/darwin_use_64_bit_inode_vs_ctypes/
+// Without it, you get the old, smaller struct stat from mac os 10.2 or so.
```

Comment:
This is specific to macOS so this needs to be in syscall_libc_darwin.go.

Reply:
Sorted.